### PR TITLE
Added changes for Helm Charts requested

### PR DESCRIPTION
**Pull request by rishumehra**
Added changes for Helm Charts requested in https://issues.redhat.com/browse/RHDEVDOCS-2082

These changes are for *OpenShift 4.6 Async*

### CODE REVIEW

File: modules/odc-installing-helm-charts.adoc [hunk 1]
```diff
@@ -19,7 +19,15 @@ image::odc_helm_chart_devcatalog.png[]
...
 .. Enter a unique name for the release in the *Release Name* field.
-.. Optionally, in the YAML editor, modify the YAML file.
-. Click *Install* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release.
+.. Select the required chart version from the *Chart Version* dorp-down.
++
+.Select chart version
+image::odc_helm_chart_select_chart_ver.png[]
++
+.. You can use the *Form view* of the *YAML view* to configure your Helm chart.
+.. Click *Install* to save the required changes and install the chart.
```
Comment (Preeticp) [inline modules/odc-installing-helm-charts.adoc:28]: I am not sure why this part of the earlier step was removed: You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release. 
The last sentence was considered a feature worth documenting for the last release.

File: modules/odc-installing-helm-charts.adoc [hunk 2]
```diff
@@ -19,7 +19,15 @@ image::odc_helm_chart_devcatalog.png[]
...
 . In the *Install Helm Chart* page:
 .. Enter a unique name for the release in the *Release Name* field.
-.. Optionally, in the YAML editor, modify the YAML file.
-. Click *Install* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release.
+.. Select the required chart version from the *Chart Version* dorp-down.
++
+.Select chart version
+image::odc_helm_chart_select_chart_ver.png[]
++
+.. You can use the *Form view* of the *YAML view* to configure your Helm chart.
```
[the remark this answers is not in the document]
Comment (Preeticp) [reply modules/odc-installing-helm-charts.adoc:27]: Add the note on Line 30 after this step, as it relates to a choic the user will make while working on this step.

File: modules/odc-installing-helm-charts.adoc [hunk 3]
```diff
@@ -19,7 +19,15 @@ image::odc_helm_chart_devcatalog.png[]
...
+.. Click *Install* to save the required changes and install the chart.
+
+*Note:*
+Some of the Helm Charts may not have the Form View. Where available, you can switch between the YAML view and Form view, the data is persisted between the views.
```
Comment (Preeticp) [inline modules/odc-installing-helm-charts.adoc:31]: Make YAML view, Form view bold.

File: modules/odc-installing-helm-charts.adoc [hunk 4]
```diff
@@ -19,15 +19,17 @@ image::odc_helm_chart_devcatalog.png[]
...
 .. Enter a unique name for the release in the *Release Name* field.
-.. Select the required chart version from the *Chart Version* dorp-down.
+.. Select the required chart version from the *Chart Version* drop-down list.
 +
 .Select chart version
 image::odc_helm_chart_select_chart_ver.png[]
 +
-.. You can use the *Form view* of the *YAML view* to configure your Helm chart.
-.. Click *Install* to save the required changes and install the chart.
+.. Use the *Form view* or the *YAML view* to configure your Helm chart.
```
Comment (Preeticp) [inline modules/odc-installing-helm-charts.adoc:27]: Please add the note on lines 30 and 31 below this bullet point. That information will be of no use to the users once he clicks install. He will be able to switch only while configuring. Add the content in the place where it is most relevant to the users and where the information to the user is most useful.

**Review by Preeticp — CHANGES_REQUESTED** [review]
@rishumehra apart from the one comment, the other changes look good to me. Add a + to the line before the note so that it aligns correctly with the previous step. You need to get a QE review and LGTM before you push to the OCP team.

File: modules/odc-installing-helm-charts.adoc (hunk 4 above)
Comment (rishumehra) [reply modules/odc-installing-helm-charts.adoc:27]: Done, incorporated the change. Please check.

**Review by rishumehra — COMMENTED** [review]
@Preeticp Please check the change

**Comment by Preeticp** [comment]
/lgtm
Thank you Rishu. Please ask @gajanan-more to review and then send it out to the OCP team for review and merge.

**Review by gajanan-more — APPROVED** [review]

File: modules/odc-installing-helm-charts.adoc [hunk 5]
```diff
@@ -19,7 +19,16 @@ image::odc_helm_chart_devcatalog.png[]
...
+image::odc_helm_chart_select_chart_ver.png[]
++
+.. Use the *Form view* or the *YAML view* to configure your Helm chart.
++
+[NOTE]
+Some of the Helm Charts may not have the *Form View* option available. Where available, you can switch between the *YAML view* and *Form view*, the data is persisted between the views.
++  
+.. Click *Install* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release.
 
 If required, you can now use the *Actions* button on the side panel or right-click the Helm release to upgrade, rollback, or uninstall the Helm release.
```
Comment (pneedle-rh) [inline modules/odc-installing-helm-charts.adoc:34]: You could minimise this sentence. For example:

"You can upgrade, rollback, or uninstall a Helm release by using the **Actions** button on the side panel or by right-clicking a Helm release."

**Review by pneedle-rh — COMMENTED** [review]
@rishumehra thank you for your contribution! I have added a few line-specific suggestions.

Can you please squash your commits into one?

Additionally, for step 1 in the 'Installing Helm charts' procedure, you could mention that there is a need to select a project after navigating to the **Add+** view. I noticed that this is required before the reader can select the **Helm Chart** option to see all the Helm Charts in the Developer Catalog. It would be useful for readers if we state this explicitly. Otherwise they could wonder where the **Helm Chart** option is after selecting the **Add+** view.

Please let me know if you have any questions.

File: modules/odc-installing-helm-charts.adoc [hunk 6]
```diff
@@ -19,7 +19,16 @@ image::odc_helm_chart_devcatalog.png[]
 +
 . In the *Install Helm Chart* page:
 .. Enter a unique name for the release in the *Release Name* field.
-.. Optionally, in the YAML editor, modify the YAML file.
-. Click *Install* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release.
+.. Select the required chart version from the *Chart Version* drop-down list.
++
+.Select chart version
+image::odc_helm_chart_select_chart_ver.png[]
```
[the remark this answers is not in the document]
Comment (rishumehra) [reply modules/odc-installing-helm-charts.adoc:25]: @pneedle-rh Agreed with your point. Will remove the screenshot from here.

**Review by rishumehra — COMMENTED** [review]
> @rishumehra thank you for your contribution! I have added a few line-specific suggestions.
> 
> Can you please squash your commits into one?
> 
> Additionally, for step 1 in the 'Installing Helm charts' procedure, you could mention that there is a need to select a project after navigating to the **Add+** view. I noticed that this is required before the reader can select the **Helm Chart** option to see all the Helm Charts in the Developer Catalog. It would be useful for readers if we state this explicitly. Otherwise they could wonder where the **Helm Chart** option is after selecting the **Add+** view.
> 
> Please let me know if you have any questions.

Thanks for notifying me the same about selecting the project in step 1, after the *+Add* view. Will incorporate that too.

**Review by rishumehra — COMMENTED** [review]
Updated the change to "+Add" and select the project before Helm Chart

**Comment by pneedle-rh** [comment]
@rishumehra this looks good to me, thank you for applying those suggestions! Can you please squash your commits into one commit? Thanks!

**Comment by rishumehra** [comment]
> @rishumehra this looks good to me, thank you for applying those suggestions! Can you please squash your commits into one commit? Thanks!

@pneedle-rh Thanks for the review, will squash the commits soon.

**Comment by pneedle-rh** [comment]
@rishumehra thanks for your time on our call today. Now that the commits have been squashed, this looks good to me! I will merge and cherry pick to 4.6 and 4.7 now.

**Pull request merged**